mscore/noteGroups.cpp

```diff
@@ -139,6 +141,8 @@ Groups NoteGroups::groups()
 void NoteGroups::resetClicked()
       {
       setSig(_sig, _groups);
+      iconPalette->setSelected(0);
+      iconPalette->update();
```

Comment:
have to force update here to trigger redraw of palette reflect resetting selecting beam mode back to the default.